### DISCUSSION TITLE
昨天晚上还很正常，今天早上打开项目发现地形部分出现了报错，有没有相同情况的朋友

### DISCUSSION THREAD

**Discussion by shaopGao** [zouyaoji/vue-cesium#404] [Q&A]
Cannot read properties of undefined (reading 'getDerivedResource')

**Comment by zouyaoji** [comment]
cesium版本更新导致的，你先用之前版本的cesium，下个版本处理一下。

**Comment by rubysiu** [comment]
哈哈哈 今天遇到同样的问题 真的就是 前几天写的代码还好好的 今天就不能用了
